backend/functions/lib/adapters/gateways/okta/okta-gateway.ts

```diff
@@ -0,0 +1,68 @@
+import { getAuthorizationConfig } from '../../../configs/authorization-configuration';
+import { Jwt, OpenIdConnectGateway } from '../../types/authorization';
+import { CamsUser } from '../../../../../../common/src/cams/session';
+import { ServerConfigError } from '../../../common-errors/server-config-error';
+import { UnauthorizedError } from '../../../common-errors/unauthorized-error';
+import { verifyAccessToken } from './HumbleVerifier';
+
+const MODULE_NAME = 'OKTA-GATEWAY';
+
+type OktaUserInfo = {
+  sub: string;
+  name: string;
+  locale: string;
+  email: string;
+  preferred_username: string;
+  given_name: string;
+  family_name: string;
+  zoneinfo: string;
+  updated_at: number;
+  email_verified: boolean;
+};
+
+async function verifyToken(token: string): Promise<Jwt> {
```

Comment:
I think it would be better if we returned just the claims from this. From a good fences rule perspective in addition to the invasive species rule, I think returning the `Jwt` which is just a copy of a type by the same name from Okta, is more than what we need and allowing the library to affect our business logic.

Reply:
Since JWT is a standards spec, we copied / "borrowed" the type definitions from Okta's codebase into our own codebase specifically because of invasive species rule. We should be importing Jwt from our copy, not from Okta dependency though so will need to double check our imports.